Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -558,29 +558,36 @@ const ScrollView = createReactClass({
   },
 
   /**
-   * Scrolls to a given x, y offset, either immediately or with a smooth animation.
+   * Scrolls to a given x, y offset, either immediately, with a smooth animation, or,
```

Comment:
Can you open a PR that applies these same changes to https://github.com/facebook/react-native-website/blob/master/docs/scrollview.md ?

These comments are not synced. At some point we need to update these comments with links to the generated docs at http://facebook.github.io/react-native instead (I've done this with a few source files already but had not gotten to ScrollView yet).